### ISSUE TITLE
Global and local `locale`

### ISSUE THREAD

**Issue by rxaviers** [kborchers/react-globalize#4]
We could allow defining `locale` globally as well. For example:

```HTML
<!-- Uses a globally defined `locale` -->
<FormatDate value={new Date()} pattern={{ date: "medium" }} />`

<!-- Uses the passed `locale` attribute -->
<FormatDate locale="en-US" value={new Date()} pattern={{ date: "medium" }} />
```

We could assume the globally defined `locale` is defined by the static `Globalize.locale()` function and use `Globalize.formatDate()` instead of `Globalize.prototype.formatDate()`.

**Comment by rxaviers** [comment]
This could be of interest https://github.com/yahoo/react-intl/blob/master/src/mixin.js#L56-L65

**Comment by kborchers** [comment]
Fixed by 4f0f173598c9349aa81bf1fd5482b2a0aee2e646
